docs/GetStarted/getting_started_linux_bazel.md

```diff
@@ -42,12 +42,12 @@ export CXX=clang++
 
 ### Clone
 
-Clone the repository and initialize its submodules:
+Clone the repository, initialize its submodules and configure:
 
 ```shell
 $ git clone https://github.com/google/iree.git
 $ cd iree
-$ git submodule update --init
+$ git submodule update --initss
```

Comment:
Is there a typo here now? (this is pending merge, can fixup after)

Reply:
ugh. for some reason vscode stopped accepting ctrl+s and I was looking everywhere for where the extra 's''s went. sorry :(